begin.html

```diff
@@ -0,0 +1,792 @@
+          <li><a href="#">Video: Arts</a></li>
+        </ul>
+        <ul>
+          <li>Living</li>
+          <li><a href="#">Automobiles</a></li>
+          <li><a href="#">Crossword</a></li>
+          <li><a href="#">Education</a></li>
+          <li><a href="#">Food</a></li>
+          <li><a href="#">Health</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#">Love</a></li>
+          <li><a href="#">Magazine</a></li>
+          <li><a href="#">Parenting</a></li>
+          <li><a href="#">Real State</a></li>
+          <li><a href="#">Recipes</a></li>
+          <li><a href="#">Style</a></li>
+          <li><a href="#">T Magazine</a></li>
+          <li><a href="#">Travel</a></li>
+        </ul>
+        <ul>
+          <li>More</li>
+          <li><a href="#">Reader Center</a></li>
+          <li><a href="#">Wirecutter</a></li>
+          <li><a href="#">Live Events</a></li>
+          <li><a href="#">The Learning Network</a></li>
+          <li><a href="#">Tools & Services</a></li>
+          <li><a href="#">N.Y.C Events Guide</a></li>
+          <li><a href="#">Multimedia</a></li>
+          <li><a href="#">Photography</a></li>
+          <li><a href="#">Video</a></li>
+          <li><a href="#">Newsletters</a></li>
+          <li><a href="#">NYT Store</a></li>
+          <li><a href="#">Times Journeys</a></li>
+          <li><a href="#">Manage My Account</a></li>
+        </ul>
+        <ul>
+          <li class="subscribe">Subscribe</li>
+          <li><a class="bold" href="#">Home Delivery</a></li>
+          <li><a class="bold" href="#">Digital Subscriptions</a></li>
+          <li><a class="bold" href="#">Crossword</a></li>
+          <li><a class="bold" href="#">Cooking</a></li>
+          <li class="space"><a href="#">Email Newsletters</a></li>
+          <li><a href="#">corporate Subsctiptions</a></li>
+          <li><a href="#">Education Rate</a></li>
+          <li><hr /></li>
+          <li><a href="#">Mobile Applications</a></li>
+          <li><a href="#">Replica Edition</a></li>
+          <li><a href="#">Español</a></li>
+          <li><a href="#">NYT Store</a></li>
+          <li><a href="#">中文网</a></li>
+        </ul>
+      </div>
+      <div class="footer-bottom">
+        <ul>
+          <li><a href="#">© 2020 The New York Times Company</a></li>
+          <li><a href="#">NYT Co</a></li>
+          <li><a href="#">Contact Us</a></li>
+          <li><a href="#">Work with us</a></li>
+          <li><a href="#">Advertise</a></li>
+          <li><a href="#">T Brand Studio</a></li>
+          <li><a href="#">Your Ad Choices</a></li>
+          <li><a href="#">Privacy</a></li>
+          <li><a href="#">Terms of Service</a></li>
+          <li><a href="#">Terms of Sale</a></li>
+          <li><a href="#">Site Map</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Subscriptions</a></li>
+        </ul>
+      </div>
+    </footer>
```

Comment:
- [ ] There is a part of the page that sticks to the bottom. Please add it.

> ![image](https://user-images.githubusercontent.com/57414789/82390898-8f6caf00-9a05-11ea-8e22-9f846a14cfbd.png)

Reply:
This requirement was done as asked.